### PR TITLE
Brazilian Portuguese Translation

### PR DESCRIPTION
**Pull request by auyer**
Brazilian Portuguese Translation related to Issue #171 

I assumed I should copy every file from the "values" folders into the new "values-pt" folders, but only the requested files were translated.

Tell me if the other files should be translated as well, like the  [arrays.xml](https://github.com/auyer/android/blob/master/libraries/cyclestreets-core/src/main/res/values-pt/arrays.xml) file, and I'll make another PR!

But I also believe other parameter files that were in the "value" folders would not need to be duplicated, depending on how the programming is made.

### CODE REVIEW

**Comment by oliverlockwood** [comment]
Hi @auyer.  Thank you for submitting the PR!  I will work with you to get this ready to go in.

As you can see from the red ❌ marked on this PR, the automated build that ran on your pull request failed, so there are some things that need correcting.  For future reference, it's always worth looking out for automated builds as you can use them to get feedback sooner than waiting for a human to write back 😃 

I will mark the things that need changing inline where I can or in follow-on comments otherwise.

File: libraries/cyclestreets-fragments/src/main/res/values-pt/dimens.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <dimen name="app_defaultsize_w">632.0dip</dimen>
+  <dimen name="app_defaultsize_h">598.0dip</dimen>
```
Comment (oliverlockwood) [inline libraries/cyclestreets-fragments/src/main/res/values-pt/dimens.xml]: This file, as well as `colors.xml`, `styles.xml`, `dimens.xml` doesn't contain text that can be localised, so there's no need to duplicate them in the `values-pt` directory.  (If the `-pt` directory doesn't contain a resource, the default is used.)

Please therefore remove any untranslated files from your PR.

**Comment by oliverlockwood** [comment]
Also @auyer I have realised that the `arrays.xml` file you mention *should* be translated - good spot!  I will correct that on the issue description.  Please would you include it as part of updating your PR?

Many thanks!

File: libraries/cyclestreets-core/src/main/res/values-pt/arrays.xml [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+    <item>Balanced</item>
+    <item>Fastest</item>
+    <item>Shortest</item>
+  </string-array>
+  <string-array name="routetype_values">
+    <item>quietest</item>
+    <item>balanced</item>
+    <item>fastest</item>
+    <item>shortest</item>
+  </string-array>
```
Comment (auyer) [inline libraries/cyclestreets-core/src/main/res/values-pt/arrays.xml]: Ok @oliverlockwood , another question : in this file, we have ```<string-array name="routetype">```  and ```<string-array name="routetype_values">``` with the same information, but one capitalized and the other not. It looks like only the first one should be translated, and the other will be used internally, is that right?

Comment (oliverlockwood) [inline libraries/cyclestreets-core/src/main/res/values-pt/arrays.xml]: @auyer You're exactly right!  You can tell we're new to this translation thing 😄 

I've just pushed a commit which marks the `_values` arrays as `untranslatable`, so you can remove those from your submission.

**Comment by auyer** [comment]
@oliverlockwood can you check if the translation is really causing the problem here? 
Cos I have checked every file, and all I believe none of the untranslatable strings still are.
I don't have a lot of practice with Travis or similar tolls (yet), but I solved the issues I could spot from 466c511  in the last commit, but the build still failed.

File: cyclestreets.vNext/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -11,7 +11,7 @@
   <string name="cyclestreets_blog_updates">Blog - updated</string>
 
   <string name="crash_notif_ticker_text">Unexpected error, please send a report …</string>
-  <string name="crash_notif_title">CycleStreets has crashed …</string>
+  <string name="crash_notif_title">CycleStets has crashed …</string>
```
Comment (oliverlockwood) [inline cyclestreets.vNext/src/main/res/values/strings.xml]: @auyer please revert the change on this line.

**Comment by auyer** [comment]
Ohh, I just saw that you merged it already. Do you still need me to fix or add anything ?

**Comment by oliverlockwood** [comment]
@auyer no worries, I just did the remaining fix myself - seemed simpler that way :-D

Thanks again for providing the translation. Do try the app if you haven't already!
